releasenotes/notes/refactor-logs-line-handler-b57e1acac51c9496.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
+  - |
+    Refactor logs/decoder/line_handler to take in content
```

Comment:
We don't need a reno note for a refacto, bare in mind that a reno note must be user friendly so we should describe what end users should expect from that change.